VMBackup/main/handle.sh

```diff
@@ -48,7 +58,7 @@ do
 	if [ -f "${cmnd}" ]
     then
 		echo "`date -u`- ${pythonVersion} path exists" >> $logfile
-		$cmnd main/handle.py -$1
+		$cmnd main/handle.py -$configSeqNo -$1
```

Comment:
although it should not matter based on the way parsing is happening in handle.py, to not change order, add $configSeqNo as second argument instead of first

Reply:
It will matter if we don't pass config seq no. 1st as we use for loop and if enable is there before configSeqNo. then it will run without config seq no. So if we want to pass enable 1st then in handle.py I need to add a for loop to take sequence no. 1st then present for loop will run for enable

Reply:
ok got it. Then keep it like this